src/app/project/project.component.html

```diff
@@ -14,7 +14,7 @@ <h1>{{ project?.name }} </h1>
     </div>
 
     <p *ngIf="builds?.length == 0" class="empty">Nothing here yet! Watching for <a
-        href="https://brigadecore.github.io/brigade/">Brigade</a> <a
+        href="https://docs.brigade.sh/topics/scripting/">Brigade</a> <a
```

Comment:
I think perhaps this link should just point to the Brigade landing page (`https://brigade.sh`) and then the link on line 18 should be updated to `https://docs.brigade.sh/topics/scripting/`.